setup.sql

```diff
@@ -34,8 +34,12 @@ CREATE TABLE cart_item (
 --wip
 CREATE TABLE orders (
 	id SERIAL PRIMARY KEY,
-	items JSON, -- will be changed later to something that can be an array of order_items
-	subtotal DECIMAL(10,2) -- for development use only, will be removed later
+	items JSON[], -- revisit for possible junction table implementation
```

Comment:
I think we should keep it as JSON for now and revisit this so it doesn't break current functionality